### PR TITLE
css-two-column-layout - Tim Silva

### PR DESCRIPTION
**Pull request by silva-tim**
Screen Shot 2023-05-18 at 2 18 31 PM

### CODE REVIEW

File: css-two-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+              deliver an unparalleled quality of care.
+            </p>
+          </div>
+        </div>
+      </header>
+      <div class="row">
+        <div class="row">
+          <div class="imgcolumn">
+            <div class="img">
+              <img src="images/software.svg" />
```
Comment (shawnkost) [inline css-two-column-layout/index.html:30]: Missing `alt` attribute

File: css-two-column-layout/index.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+              We build products that transform how healthcare facilities run
+              across Africa<br />
+              - making them radically more efficient operationally and
+              financially, a complete turnaround.
+            </p>
+          </div>
+          <div class="middle"></div>
+          <div class="imgcolumn">
+            <div class="img">
+              <img src="images/practice.svg" />
```
Comment (shawnkost) [inline css-two-column-layout/index.html:48]: Missing `alt` attribute

File: css-two-column-layout/index.html [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+              Our all-in one electronic health records (EHR), hospital
+              management (HMIS) &amp; biling system helps streamline your
+              facility, reduce waste and improve quality of care.
+            </p>
+          </div>
+        </div>
+        <div class="row">
+          <div class="imgcolumn">
+            <div class="img">
+              <img src="images/circumstance.svg" />
```
Comment (shawnkost) [inline css-two-column-layout/index.html:63]: Missing `alt` attribute

File: css-two-column-layout/index.html [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+            <p>
+              Our intuitive system is built around the peculiarities of your
+              facility; we support offline deployments, sync to the cloud and
+              run on any device 24/7.
+            </p>
+          </div>
+          <div class="middle"></div>
+          <div class="imgcolumn">
+            <div class="img">
+              <img src="images/data.svg" />
```
Comment (shawnkost) [inline css-two-column-layout/index.html:77]: Missing `alt` attribute

**Review by shawnkost — CHANGES_REQUESTED** [review]
Missing a few `alt` attributes here!

File: css-two-column-layout/index.html (hunk 4 above)
Comment (silva-tim) [reply css-two-column-layout/index.html]: Added alt tags

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**